templates/ratecard-rates.html.twig

```diff
@@ -0,0 +1,27 @@
+{#
+/**
+ * @file
+ * Default theme implementation to display a apigee rate plan detail.
+ *
+ * Variables:
+ * - detail:      See: "rate-plan-detail.html.twig"
+ * - rates:
+ *   - id:                          The unique ID of the rate plan rate.
+ *   - startUnit:                   The number the rate starts at.
+ *   - endUnit:                     The rate is good until this nuber of units.
+ *   - rate:                        The rate that is applies within this range.
+ */
+#}
+
+<div class="rate_plan__detail-card__ratecard-rates rate-list-wrapper">
+    <div class="row ratecard-rate__header row">
+        <div class="column"><strong>{{ "Volume bands"|t }}</strong></div>
+        <div class="column"><strong>{{ "Cost per API call"|t }}</strong></div>
+    </div>
+    {% for rate in rates %}
+        <div class="row ratecard-rate">
+            <div class="column">{{ "Greater than"|t }} {{ rate.startUnit }} {% if rate.endUnit %}{{ "up to"|t }} {{ rate.endUnit }}{% endif %}</div>
+            <div class="column">{{ rate.rate }}</div>
```

Comment:
We definitely do not want to use `.row` and `.column`.   I still think these are best displayed in a table, since they are tabular data, but if you really want to add row/column CSS, these need to be prefixed.  `rate-plan-rates__row` and `rate-plan-rates__column`.

Reply:
I changed these to `rate-plan-rates__row` and `rate-plan-rates__column`.